lib/ts/recipe/withOverride.tsx

```diff
@@ -0,0 +1,17 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/react";
+import React, { useContext } from "react";
+import { ComponentOverrideContext } from "./componentOverrideContext";
+
+export const withOverride = <TComponent extends React.FunctionComponent<any>>(
```

Comment:
Yes, it will work. This is because `withOverride` returns an anonymous function component that accepts original component's props typed as `React.ComponentProps<TComponent>`

Reply:
Please feel free to resolve this.